msr_rr_cfp.md

```diff
@@ -0,0 +1,83 @@
+# Title (required)
+Provide the working title of your study. It may be the same title that you submit for publication of your final manuscript, but it is not a requirement.
+* **Example**:: Should your family travel with you on the enterprise?
+* Subtitle (optional): Effect of accompanying families on the work habits of crew members
+
+# Authors (required)
+At this stage, we believe that an unblinded/single blind review is most productive
```

Comment:
How would this play out if the final study is then submitted to a DBR venue?

Reply:
well in this case, it will go to EMSE, which is single blind. But the RR format on open science foundation does have an embargo option.